bookstore/tests/functional/test_review_form_submission.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from django.test import Client
+
+
+@pytest.mark.django_db
+def test_form_submission(test_book):
```

Comment:
The form itself in main code is not working - the issue has been already created. The test case will not work until the issue is resolved.